socorro/signature/rules.py

```diff
@@ -240,26 +244,23 @@ def normalize_frame(
                 file = filename.rsplit("/")[-1]
             return f"{file}#{line}"
 
-        # If there's an offset and no module/module_offset, use that
-        if not module and not module_offset:
-            if unloaded_modules:
-                # Use the first unloaded module and the offset or "0"
-                unloaded_module = unloaded_modules[0]
-                unloaded_module_module = unloaded_module.get("module")
-                unloaded_module_offsets = unloaded_module.get("offsets")
-                if unloaded_module_module and unloaded_module_offsets:
-                    return "(unloaded {}@{})".format(
-                        unloaded_module_module,
-                        strip_leading_zeros(unloaded_module_offsets[0]),
-                    )
-                elif unloaded_module_module:
-                    return "(unloaded {})".format(unloaded_module_module)
+        # If there's a module, use that
+        if module:
+            return module
+
+        # If there are unloaded modules, use the first one
+        if unloaded_modules:
+            unloaded_module = unloaded_modules[0]
+            unloaded_module_module = unloaded_module.get("module")
+            return f"(unloaded {unloaded_module_module})"
 
-            if offset:
-                return f"@{strip_leading_zeros(offset)}"
+        # If there's only an offset, then begrudgingly use that
+        if offset:
+            return f"@{strip_leading_zeros(offset)}"
```

Comment:
We keep the offset here because otherwise there isn't anything in this resulting frame signature. This either doesn't come up much or doesn't come up at all. I wasn't able to find a crash report that hits this scenario. Maybe it still happens but rarely? Maybe I was searching poorly?

I figured we can think about this more another day.